minio/operator
# Remove namespaces from Operator Helm chart

## Description
The PR removes redundant `namespace` from Operator Helm chart templates. Similar to https://github.com/minio/minio/pull/16968.
_Note: There is no `namespace` properties in Tenant Helm chart templates._

## Motivation and Context
Motivation - keep code smaller and cleaner.
Context - Helm creates Kubernetes resources (Deployment, ConfigMap, Service, Ingress etc.) in the same namespace where release installed, i.e. `.Release.Namespace` is used by default.

## How to test this PR?
Install the chart to `qwerty` namespace:
```shell
$ helm install test helm/operator -n qwerty
NAME: test
LAST DEPLOYED: Sat Jun 17 00:09:16 2023
NAMESPACE: qwerty
STATUS: deployed
REVISION: 1
TEST SUITE: None
NOTES:
   ...
```
Check reqources in the `qwerty` namespace:
```
$ kubectl get all -n qwerty
NAME                                  READY   STATUS    RESTARTS   AGE
pod/console-869647699-rm242           1/1     Running   0          37s
pod/minio-operator-6f47c94d76-46jpg   1/1     Running   0          37s
pod/minio-operator-6f47c94d76-dgtd2   1/1     Running   0          37s

NAME               TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)             AGE
service/console    ClusterIP   10.99.126.7     <none>        9090/TCP,9443/TCP   37s
service/operator   ClusterIP   10.100.122.50   <none>        4221/TCP            37s
service/sts        ClusterIP   10.98.185.5     <none>        4223/TCP            37s

NAME                             READY   UP-TO-DATE   AVAILABLE   AGE
deployment.apps/console          1/1     1            1           37s
deployment.apps/minio-operator   2/2     2            2           37s

NAME                                        DESIRED   CURRENT   READY   AGE
replicaset.apps/console-869647699           1         1         1       37s
replicaset.apps/minio-operator-6f47c94d76   2         2         2       37s

```
